docker.sh

```diff
@@ -6,24 +6,12 @@ GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 IMAGE_NAME=rustvmm/dev
 REGISTRY=index.docker.io
 
-# Get the latest published version. Returns a number.
-# If latest is v100, returns 100.
-# If latest for riscv64 is v100-riscv, returns 100.
-# This works as long as we have less than 100 tags because we set the page size to 100,
-# once we have more than that this script needs to be updated.
-latest(){
-  curl -L -s 'https://registry.hub.docker.com/v2/repositories/rustvmm/dev/tags?page_size=100'| \
-    jq '."results"[]["name"]' |  sed 's/"//g' | cut -c 2- | grep -E "^[0-9]+" | sort -n | tail -1
-}
-
 next_version() {
-    latest_version=$(latest)
-    new_version=$((latest_version + 1))
-    echo "$new_version"
+    echo "$(git show -s --format=%h)"
 }
 
 print_next_version() {
-  echo "${IMAGE_NAME}:v${next_version}"
+  echo "${IMAGE_NAME}:${next_version}"
```

Comment:
I would prepend `g` to the sha1 like in the Linux Kernel:

```suggestion
  echo "${IMAGE_NAME}:g${next_version}"
```